tools/dynamic_framework_slicer/README

```diff
@@ -0,0 +1,5 @@
+dynamic_framework_slicer takes a directory containing .app bundle structures,
+iterates over the frameworks in the .app, and ensures that each framework only
+contains the same slices as the app binary.
+
+bundletool can run on Darwin only.
```

Comment:
bundletool?

Reply:
I didn't copy this from anywhere.... *cough* :)